app/assets/stylesheets/application.css.scss

```diff
@@ -14,6 +14,80 @@
  *= require_self
  */
 
+ #gif-overlay {
+   background-color: rgba(10, 10, 10, 0.5);
+   background-size: cover;
+   position: absolute;
+   width:  100vw;
+   height: 100vh;
+ }
+
+ #gif-container {
+  width:  100vw;
+  height: 100vh;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: fixed;
+  background-image: url(http://i.imgur.com/WTrMmzJ.jpg);
```

Comment:
URL `http://i.imgur.com/WTrMmzJ.jpg` should not contain protocol or domain<br>URLs should be enclosed in quotes